content/millennium/r4/base/entities.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Entities | R4 API
+layout: api
+---
+
+# Entities
+
+<%= render '/millennium/r4/base/entities.*' %>
+
+* [Organization](../base/entities/organization)
+* [Location](../base/entities/location)
```

Comment:
I think we're missing the `[Device]` item on this list

Reply:
https://github.com/cerner/fhir.cerner.com/commit/775cdd410a50bbc809a19a30052924b93f1fd20f